src/pages/JammerPage.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import Accordion from "../components/Accordion";
+import db from "../db.json";
+import {
+  CenteredParagraph,
+  Heading,
+  PageContainer,
+  Paragraph,
+  StyledCard,
+  SubHeading,
+  Title,
+} from "../styles/sharedStyles";
+import { JudgingContainer, PrizesContainer } from "./JammerPage.styled";
+
+export const JammerPage: React.FC = () => {
+  const jammer = db.jammers[0];
+
+  return (
+    <PageContainer>
+      <>
+        <Heading>{jammer.welcome}</Heading>
+        <Paragraph>{jammer.welcomeDescription}</Paragraph>
+
+        <Title>{jammer.challengeTitle}</Title>
+        <Paragraph>{jammer.challengeDescription}</Paragraph>
+        <StyledCard>
+          <SubHeading>{jammer.internalColleagues}</SubHeading>
+          <CenteredParagraph>{jammer.challengeInternal}</CenteredParagraph>
+        </StyledCard>
+
+        <StyledCard>
+          <SubHeading>{jammer.externalColleagues}</SubHeading>
+          <CenteredParagraph>{jammer.challengeExternal}</CenteredParagraph>
+        </StyledCard>
+        <Title>{jammer.judgingTitle}</Title>
+        <CenteredParagraph>{jammer.judgingDescription}</CenteredParagraph>
+        <JudgingContainer>
+          {jammer.judgingCriteria.criteria.map((criteria, index) => (
+            <Accordion key={index} title={criteria.title}>
```

Comment:
A small point which doesn't matter here because the array is fixed, but React doesn't encourage using array indexes as keys because there can be errors if an element is inserted or deleted or if the array is reordered. https://react.dev/learn/rendering-lists#why-does-react-need-keys (the "Pitfall" section). I did this myself in Table.tsx and I should fix it. Better to use some data as the key, like `<Accordion key={criteria.title} title={criteria.title}>`. But since the data is fixed here, here it is fine. Just worth keeping in mind.